features/03_testing_frameworks/cucumber/steps/command/check_output_of_command.feature

```diff
@@ -41,6 +41,28 @@ Feature: All output of commands which were executed
     When I run `cucumber`
     Then the features should all pass
 
+  Scenario: Failed detection of one-line output
+    Given an executable named "bin/aruba-test-cli" with:
+    """bash
+    #!/usr/bin/env bash
+
+    echo 'hello world'
+    """
+    And a file named "features/output.feature" with:
+    """cucumber
+    Feature: Run command
+      Scenario: Run command
+        When I run `aruba-test-cli`
+        Then the output should contain "goodbye world"
+    """
+    When I run `cucumber`
+    Then the features should not all pass with:
+    """
+          expected `aruba-test-cli` to have output string includes: "goodbye world"
```

Comment:
**Narrator:** _I was missing something_.

Perhaps I am missing something, but line 61 and line 56 have mismatched wording.

>  +          expected `aruba-test-cli` to have output string includes: "goodbye world"

"to have output string _including_" would be my personal expectation (just reading the line)